app/routes/cdo/create/dog-details.js

```diff
@@ -56,7 +56,8 @@ const validatePayload = (payload) => {
       'any.required': 'CDO issue date must include a valid year',
       'number.empty': 'CDO issue date must include a valid year',
       'number.base': 'CDO issue date must include a valid year'
-    })
```

Comment:
Were we going to add validation that the CDO issue date shouldn't be in the future?

Reply:
I was going to do that on a different branch but will fix here.